src/common/utils/sleep.ts

```diff
@@ -0,0 +1,5 @@
+export function sleep(ms: number) {
```

Comment:
nit: add return type of the function..

Reply:
Although typescript automatically infers return types i think enforcing explicit return types would still be beneficial. One benefit i found out is; we can reduce typescript compile times using explicit return types.  

I will look for a way to lint this.